rules/cloud/m365/audit/microsoft365_teams_guest_rmm_deployment.yml

```diff
@@ -0,0 +1,56 @@
+title: Detection of Potential Malicious Guest Accounts and RMM Tool Deployment via Teams
+id: aed9c24f-097a-4505-af85-74b4b83982b0
+status: experimental
+description: Detects potential malicious guest accounts using onmicrosoft.com and deployment of RMM tools via Teams messages.
```

Comment:
This doesn't really makes a lot of sense, can you please rephrase it. What's the relation between guest accounts and deployment of RMMs?

Reply:
As I mentioned in the notes below, attackers are using Teams channels to send Remote Monitoring and Management (RMM) software, posing as fake Microsoft support agents and asking end users to download these RMM tools under the guise of troubleshooting assistance.